charts/dgraph/values.yaml

```diff
@@ -472,7 +472,7 @@ ratel:
   image:
     registry: docker.io
     repository: dgraph/ratel
-    tag: v21.03.1
+    tag: v21.12.0
```

Comment:
This looks like a regression (21.12 is unsupported). Should this be v23?

Reply:
@damonfeldman There is no v23 for Ratel.  See https://hub.docker.com/r/dgraph/ratel/tags.  I picked the latest tag. 
Were there changes made in Ratel master that are needed for Dgraph v23?